packages/plugins/src/chainflip/broker.ts

```diff
@@ -47,7 +47,7 @@ const registerAsBroker = (toolbox: Awaited<ReturnType<typeof ChainflipToolbox>>)
     throw new SwapKitError("chainflip_broker_register");
   }
 
-  return toolbox.signAndBroadcast(extrinsic);
+  return toolbox.signAndBroadcast({ tx: extrinsic, address: toolbox.getAddress() });
```

Comment:
That could be retrieved at toolbox level

Reply:
hmm yes could be. I think we currently do not offer that get address logic in there. should be changed on toolbox level -> wallets overwrite

similar to EVM

👍